src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -710,7 +709,6 @@ private JabRefPreferences() {
         defaults.put(WARN_BEFORE_OVERWRITING_KEY, Boolean.TRUE);
         defaults.put(CONFIRM_DELETE, Boolean.TRUE);
         defaults.put(CONFIRM_LINKED_FILE_DELETE, Boolean.TRUE);
-        defaults.put(TRASH_INSTEAD_OF_DELETE, JabRefDesktop.moveToTrashSupported());
```

Comment:
Shouldn't there a `true` be put?

OK, what if the OS does not support move to trash?

This were my thoughts. Therefore, I made the default rely on the current OS's capabilities.

Reply:
Exactly this line broke the menu bar!